sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -11,7 +11,11 @@
 
 ### Bug fixes
 
-- [Bug Fix] `expiresAtUtc` is `Invalid Date` in the received message when the ttl is not defined. Has been fixed in [#13543](https://github.com/Azure/azure-sdk-for-js/pull/13543)
```

Comment:
Wondering why this changelog entry was removed. Was there a merge conflict?

Reply:
Oh no, I'll revert. Thanks! 🙂